CHANGELOG.md

```diff
@@ -376,11 +385,12 @@ fix(events): change type expected in json parsing from implicit int to explicit
 
 * Add region support [#129](https://github.com/Scalingo/go-scalingo/pull/129)
 This comes with the following changes:
- - Removed the `NewClient` method. You should use `New` instead
- - Add the `Region` field to `ClientConfig`
- - Drop support for the `SCALINGO_AUTH_URL`, `SCALINGO_API_URL`, `SCALINGO_DB_URL` environment variables. You should use the corresponding `ClientConfig` fields instead.
- - Remove the default values for `APIEndpoint` and `DatabaseAPIEndpoint`
- - Add some basic examples
+
+* Removed the `NewClient` method. You should use `New` instead
+* Add the `Region` field to `ClientConfig`
+* Drop support for the `SCALINGO_AUTH_URL`, `SCALINGO_API_URL`, `SCALINGO_DB_URL` environment variables. You should use the corresponding `ClientConfig` fields instead.
+* Remove the default values for `APIEndpoint` and `DatabaseAPIEndpoint`
+* Add some basic examples
```

Comment:
This change is not correct. These items are supposed to be sub-items of the item in lines 386-387